src/olympia/abuse/management/commands/fill_cinderjobs_denormalized_fields.py

```diff
@@ -0,0 +1,33 @@
+from django.core.management.base import BaseCommand
+
+from olympia.abuse.models import CinderJob
+from olympia.addons.models import Addon
+
+
+class Command(BaseCommand):
```

Comment:
Would the plan be to remove this code after running it in prod? Or would we need this command later? Feels like adding code that will just hang around without a plan to remove it.